client/domain/FeedbackObjectFactory.js

```diff
@@ -22,6 +22,10 @@ tie.factory('FeedbackObjectFactory', [
     var Feedback = function(answerIsCorrect) {
       this._paragraphs = [];
       this._answerIsCorrect = answerIsCorrect;
+      // This records what message was displayed with this feedback.
+      // If no message (or no message with additional potential messages)
+      // was displayed, this will remain at -1.
```

Comment:
In general, null is like None in python, which is a fine value for ints/floats/strings/etc. properties that indicates the absence of a value. So it's totally fine to use it for fields of any given type. (Otherwise you wouldn't ever be able to use it...)

So I think null is fine; -1 seems a bit opaque to me and doesn't necessarily convey "none".